dark/index.html

```diff
@@ -239,7 +239,7 @@ <h3>CONTACT <span class="text-info">ME</span></h3>
           just below.
         </p>
         <p class="contact-text">
-          <i class="fa fa-envelope"></i>Email (Personal) : yoshithasbrathnayake@gmail.com

```

Comment:
Don't remove the `fa fa-envelope` bcs it is the email icon @Rishav1707

Reply:
Done